static/app/views/performance/transactionSummary/transactionOverview/suspectSpans.tsx

```diff
@@ -0,0 +1,302 @@
+import SortLink from 'app/components/gridEditable/sortLink';
+import Link from 'app/components/links/link';
+import Pagination, {CursorHandler} from 'app/components/pagination';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import {defined} from 'app/utils';
+import EventView from 'app/utils/discover/eventView';
+import {getFieldRenderer} from 'app/utils/discover/fieldRenderers';
+import {ColumnType, fieldAlignment} from 'app/utils/discover/fields';
+import SuspectSpansQuery from 'app/utils/performance/suspectSpans/suspectSpansQuery';
+import {decodeScalar} from 'app/utils/queryString';
+
+import {SpanSortOthers, SpanSortPercentiles} from '../transactionSpans/types';
+import {
+  getSuspectSpanSortFromLocation,
+  spansRouteWithQuery,
+} from '../transactionSpans/utils';
+import {generateTransactionLink} from '../utils';
+
+type SuspectSpanTableColumnKeys =
+  | 'op'
+  | 'description'
+  | 'p75ExclusiveTime'
+  | 'count'
+  | 'sumExclusiveTime'
+  | 'id'
+  | 'spans';
+
+type SuspectSpanTableColumn = GridColumnOrder<SuspectSpanTableColumnKeys>;
+
+type SuspectSpanTableDataRow = Record<SuspectSpanTableColumnKeys, any>;
+
+const SUSPECT_SPANS_TABLE_COLUMN_ORDER: SuspectSpanTableColumn[] = [
+  {
+    key: 'op',
+    name: t('Operation'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: 'description',
+    name: t('Description'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: 'p75ExclusiveTime',
+    name: t('P75 Exclusive Time'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: 'count',
+    name: t('Total Occurrences'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: 'sumExclusiveTime',
+    name: t('Total Exclusive Time'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: 'id',
+    name: t('Example Transaction'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+];
+
+const SPANS_CURSOR_NAME = 'spansCursor';
+
+type Props = {
+  location: Location;
+  organization: Organization;
+  eventView: EventView;
+  projectId: string;
+  transactionName: string;
+};
+
+export default function SuspectSpans(props: Props) {
+  const {location, organization, eventView, projectId, transactionName} = props;
+  const sort = getSuspectSpanSortFromLocation(location, 'spanSort');
+  const cursor = decodeScalar(location.query?.[SPANS_CURSOR_NAME]);
+
+  const sortedEventView = eventView
+    .withColumns(
+      [...Object.values(SpanSortOthers), ...Object.values(SpanSortPercentiles)].map(
+        field => ({kind: 'field', field})
+      )
+    )
+    .withSorts([{kind: 'desc', field: sort.field}]);
+  const sortColumn: GridColumnSortBy<SuspectSpanTableColumnKeys> = {
+    key: sort.field as SuspectSpanTableColumnKeys,
+    width: COL_WIDTH_UNDEFINED,
+    order: 'desc',
+  };
+
+  return (
+    <SuspectSpansQuery
+      location={location}
+      orgSlug={organization.slug}
+      eventView={sortedEventView}
+      perSuspect={1}
+      cursor={cursor}
```

Comment:
what, no `0:0:1`? lol

Reply:
No need for that this time since we want this to paginate, just independently from the other parts of the page 🤣